docs/index.md

```diff
@@ -953,8 +953,8 @@ INFO[0008] Starting machine back up...
 INFO[0008] Waiting for VM to start...
 ```
 
-> **Note**: If you are using a custom boot2docker ISO specified using
-> `--virtualbox-boot2docker-url` or an equivalent flag, running an upgrade on
+> **Note**: If you are using a custom VirtualBox ISO specified using
```

Comment:
This isn't _strictly_ true any more as it is handled by the provisioner, I wonder if / when we should update it.